ui/packages/catalog-ui-search/src/main/webapp/component/announcement/announcementsError.tsx

```diff
@@ -0,0 +1,5 @@
+class AnnouncementsError extends React.Component {
```

Comment:
❓ What is the purpose of this component? It doesn't look like you use it.

✏️ It also isn't being exported

Reply:
Got rid of it, thank you ser